platform/android/java/build.gradle

```diff
@@ -92,15 +94,20 @@ def templateExcludedBuildTask() {
 /**
  * Generates the build tasks for the given flavor
  * @param flavor Must be one of the supported flavors ('template' / 'editor')
+ * @param editorVendor Must be one of the supported editor vendors ('google' / 'meta')
```

Comment:
Does the `google` build have anything specific to the Play Store? In my understanding it should be generic and run on any Android/AOSP device when installed directly as an APK, or could possibly be shipped on non-Google stores too, no?

If so it might make sense to make the options `generic`/`meta`? Or are we using `google` for Android/AOSP and not for Google Play?

Reply:
yeah, we're using `google` for Android/AOSP.